### ISSUE TITLE
ADD 503 SIP SIGNAL DETAILS + STATUS TO LIVEKIT WEBHOOK EVENT

### ISSUE THREAD

**Issue by AutoScrape123TX** [livekit/sip#191]
Please add the catch-all SIP signal (503) to the livekit event SIP status "participant_left".

This way we could know directly in webhook events the sip call got errors.

**Comment by dennwc** [comment]
Good idea, will map SIP errors to a separate participant leave status.
